src/components/product-parameter/ProductParameter.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import LabelledInput from '../labelled-input'
+
+const ProductParameter = ({ text, options, onChange, ...other }) => {
+  const [defaultValue, setDefaultValue] = useState('Выбрать')
+  const [option, setOption] = useState({ name: 'my name', value: 'my value' })
+
+  const selectInput = (
+    <select
+      name={text}
+      onChange={() => {
+        setDefaultValue(null)
+        setOption({ value: 'Hi', name: 'Bye' })
```

Comment:
I believe you meant to call `onChange` here with `event.target.value` And thus we need to add `event` to the arguments of arrow function